src/site/_filters/slugify.js

```diff
@@ -19,12 +19,12 @@ const slugify = require('slugify');
 /**
  * Slugifies string.
  *
- * @param {string | String} str
+ * @param {string|StringConstructor} str
```

Comment:
Does this actually happen in practice? I've always been scared of the upper-case `String` type.

Reply:
I'm unsure if we ever use String objects instead of primitives, but if we were this would be the correct typing (from what I could tell).